test/blocks/global-navigation/utilities/utilities.test.js

```diff
@@ -26,27 +28,69 @@ describe('global navigation utilities', () => {
     expect(fragment2.tagName).to.equal('SPAN');
   });
 
+  // TODO - no tests for using the the live url and .hlx. urls
+  // as mocking window.location.origin is not possible
+  describe('getFedsContentRoot', () => {
+    it('should return content source for localhost', () => {
+      const contentSource = getFedsContentRoot();
+      expect(contentSource).to.equal('https://main--milo--adobecom.hlx.page');
+    });
+  });
+
+  describe('federatePictureSources', () => {
+    it('should use federated content root for image sources', async () => {
+      const imgPath = 'test/blocks/global-navigation/mocks/media_medium_dropdown.png';
+      const template = toFragment`<picture>
+          <source
+            type="image/webp"
+            srcset="./${imgPath}"
+            media="(min-width: 600px)"/>
+          <source
+            type="image/webp"
+            srcset="./${imgPath}"/>
+          <source
+            type="image/png"
+            srcset="/${imgPath}"
+            media="(min-width: 600px)"/>
+          <img
+            loading="lazy"
+            alt=""
+            type="image/png"
+            src="/${imgPath}"/>
+        </picture>`;
+
+      federatePictureSources(template);
+      document.querySelectorAll('source, img').forEach((source) => {
+        const attr = source.hasAttribute('src') ? 'src' : 'srcset';
+        expect(source.getAttribute(attr) === `https://main--milo--adobecom.hlx.page/${imgPath}`);
```

Comment:
Your expectation is not correct:
```suggestion
          expect(source.getAttribute(attr)).to.equal(`https://main--milo--adobecom.hlx.page/${imgPath}`);
```

Reply:
Good catch, forgot a `.to.be.true` 🙈